core/templates/dev/head/domain/topic/TopicUpdateService.js

```diff
@@ -195,13 +195,21 @@ oppia.factory('TopicUpdateService', [
        *    created in the current draft of the topic. (i.e, the subtopic to
        *    delete hasn't been saved in the datastore yet.)
        */
-      deleteSubtopic: function(topic, subtopicId, isNewlyCreated) {
+      deleteSubtopic: function(topic, subtopicId) {
         var subtopic = topic.getSubtopicById(subtopicId);
         if (!subtopic) {
           throw Error('Subtopic doesn\'t exist');
         }
         var title = subtopic.getTitle();
         var skillIds = subtopic.getSkillIds();
+        var isNewlyCreated = false;
```

Comment:
this is a variable name, don't start with "is"

Reply:
Ok, done.